gto/git_utils.py

```diff
@@ -270,7 +283,7 @@ def git_push(repo_path: str) -> None:
 
 
 def git_add_and_commit_all_changes(repo_path: str, message: str) -> None:
```

Comment:
need to update the signature of this function then, `repo_path: str` should probably change into `repo: Union[str, git.Repo]`?